hoot-services/src/main/java/hoot/services/controllers/export/AlphaShapeCommand.java

```diff
@@ -53,13 +53,17 @@
 
         options.add("writer.precision=7");
 
-        // bounding box is optional for this command; if not specified, the
+        // bounding area is optional for this command; if not specified, the
         // command will calculate for the combined extent of all input datasets which, of course, can
         // be very expensive for large datasets
-        if (params.getBounds() != null) {
-            BoundingBox bounds = new BoundingBox(params.getBounds());
-            options.add("convert.bounds=" + bounds.getMinLon() + "," + bounds.getMinLat() + ","
-                    + bounds.getMaxLon() + "," + bounds.getMaxLat());
+        String bounds = params.getBounds();
+        if (bounds != null) {
+            if (!bounds.contains(";")) {
+                BoundingBox bbox = new BoundingBox(bounds);
+                bounds = bbox.getMinLon() + "," + bbox.getMinLat() + "," + bbox.getMaxLon() + "," + bbox.getMaxLat();
+            }
+
+            options.add("convert.bounds=" + bounds);
```

Comment:
So we assume here the if the bounds param is not a poly with semicolons, it can be used in `convert.bounds` as-is?
I thought `convert.bounds` could accept a poly now and would not need to convert it to a bounding box?
Same comment with calc tiles below.

Reply:
yea you're right but I figured we leave the `new BoundingBox` logic as a way to also validate the coords they send?

Reply:
I'm just not sure if we get a better result by passing the poly to these commands vs the poly extent

Reply:
I missed the `!` in the conditional.  So a bbox string we validate by passing to a BoundingBox constructor, but for a poly parameter we just pass as-is.

Reply:
correct. too hard to validate since polys can be very odd shapes